tests/e2e/br/br.go

```diff
@@ -1127,7 +1127,6 @@ var _ = ginkgo.Describe("Backup and Restore", func() {
 			err = createRestoreAndWaitForComplete(f, restoreName, backupClusterName, typ, logBackupName, func(restore *v1alpha1.Restore) {
 				restore.Spec.Mode = v1alpha1.RestoreModePiTR
 				restore.Spec.PitrFullBackupStorageProvider.S3 = fullBackup.Spec.S3
-				restore.Spec.PitrRestoredTs = currentTS
```

Comment:
can you also add one more case with the PitrRestoredTs set to a previous point of time?